application/wso2verify/app/src/main/java/org/wso2/carbon/identity/mobile/wso2verify/AuthenticateActivity.kt

```diff
@@ -0,0 +1,97 @@
+package org.wso2.carbon.identity.mobile.wso2verify
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import androidx.fragment.app.FragmentActivity
+import java.util.concurrent.Executors
+import android.widget.TextView
+import androidx.biometric.BiometricPrompt
+import com.google.firebase.iid.FirebaseInstanceId
+import kotlinx.android.synthetic.main.activity_authenticate.*
```

Comment:
remove * imports, do the same in all other places

Reply:
Done